src/manifold/src/sort.cpp

```diff
@@ -256,21 +231,33 @@ void Manifold::Impl::Finish() {
                "Not an even number of faces after sorting faces!");
 
 #ifdef MANIFOLD_DEBUG
+  auto MaxOrMinus = [](int a, int b) {
+    return std::min(a, b) < 0 ? -1 : std::max(a, b);
+  };
+  int face = 0;
   Halfedge extrema = {0, 0, 0, 0};
-  extrema = reduce(halfedge_.begin(), halfedge_.end(), extrema, Extrema());
-#endif
-
+  for (size_t i = 0; i < halfedge_.size(); i++) {
```

Comment:
Simplified the code a bit as well. Was thinking about using pair iterators, but it is too much work. We don't need parallelization here anyway.